src/main/cli/cli.c

```diff
@@ -1252,25 +1253,51 @@ static int cliParseSerialMode(const char *tok)
     return mode;
 }
 
+static int cliParseSerialOptions(const char *tok)
+{
+    int options = 0;
+
+    if (strcasestr(tok, "bidir")) {
+        options |= SERIAL_BIDIR;
+    }
+    if (strcasestr(tok, "bidir_pp")) {
+        options |= SERIAL_BIDIR_PP;
+    }
+    if (strcasestr(tok, "inverted")) {
+        options |=  SERIAL_INVERTED;
+    }
+    if(options == 0) {
+        options = SERIAL_NOT_INVERTED;
```

Comment:
`SERIAL_NOT_INVERTED` is zero. The macro exists for documentation purposes, using it this way is unnecessary.

Reply:
ok, didnt know. will remove it on next commit

Reply:
done

Reply:
i completly removed the options stuff now. please see my comment below